src/encode/pattern/mod.rs

```diff
@@ -584,14 +584,16 @@ impl FormattedChunk {
                     }
                     Level::Warn => w.set_style(Style::new().text(Color::Yellow))?,
                     Level::Info => w.set_style(Style::new().text(Color::Green))?,
-                    Level::Trace => w.set_style(Style::new().text(Color::Black).intense(true))?,
```

Comment:
you could probably remove the .intense or the branch altogether I think

Reply:
Eh, I'd like to keep the intense so it's more clear it's a logging level.

Reply:
I would prefer to have us in parity with [`env_logger`](https://github.com/env-logger-rs/env_logger/blob/cf5d6061ffec2c90aa64ffef1de8fbba33e3aaa7/src/fmt/writer/termcolor/extern_impl.rs#L48).

Reply:
If I'm understanding correctly, the color in env_logger causes issues, at least on my terminal.

Reply:
Wait nevermind, it has cyan.

Reply:
Huh, in the source code of env_logger it does set the default color of trace to Cyan, but I just can't get it to print Cyan, but every other color is the same as what's printed. I distinctly remember seeing something very similar to (Color::Black).intense(true) when I originally made the PR and was the reason why I used intense here, but I just can't find it in their source code right now.